src/CLI/clitree/cli-xml/ipv4-neighbors.xml

```diff
@@ -62,29 +62,161 @@ limitations under the License.
     <COMMAND
         help="ARP entries for this interface"
         name="show ip arp interface">
-        <PARAM
-            name="if-type"
-            help=""
-            ptype="INTF_TYPE">
-            <PARAM
-                name="if-id"
-                help="Numeric ID for the given interface type"
-                ptype="UINT">
-                <PARAM
-                    help="Summary of ARP entries"
-                    name="summary"
-                    ptype="SUBCOMMAND"
-                    mode="subcommand"
-                    optional="true">
+        <PARAM 
+            name="iftype" 
+            help="interface type" 
+            mode="switch" 
+            ptype="SUBCOMMAND">
+            <PARAM 
+                name="Ethernet" 
+                help="Select an interface" 
+                mode="subcommand" 
+                ptype="SUBCOMMAND">
+                <PARAM 
+                    name="phy-if-name" 
+                    help="Physical interface" 
+                    ptype="UINT">
+                    <PARAM
+                        help="Summary of ARP entries"
+                        name="summary"
+                        ptype="SUBCOMMAND"
+                        mode="subcommand"
+                        optional="true">
+                    </PARAM>
                 </PARAM>
             </PARAM>
+            <PARAM 
+                name="Loopback" 
+                help="Loopback Interface Configuration" 
+                mode="subcommand" 
+                ptype="SUBCOMMAND">
+                <PARAM 
```

Comment:
please remove loopback, mgmt and vxlan intfs.

Reply:
@stalabi1  - Is this done ?

Reply:
yes